src/query/storages/parquet/src/parquet2/parquet_reader/reader.rs

```diff
@@ -237,12 +238,12 @@ impl Parquet2Reader {
         Ok(Box::new(OneBlock(Some(block))))
     }
 
-    pub fn read_from_readers(&self, readers: &mut IndexedReaders) -> Result<Vec<IndexedChunk>> {
+    pub fn read_from_merge_io(&self, readers: &mut IndexedChunks) -> Result<Vec<IndexedChunk>> {
```

Comment:
IndexedReaders is replaced with IndexedChunks.
but `readers/reader` is still used a lot.